src/webui/src/components/trial-detail/Intermediate.tsx

```diff
@@ -226,62 +231,64 @@ class Intermediate extends React.Component<IntermediateProps, IntermediateState>
         }
     }
 
-    shouldComponentUpdate(nextProps: IntermediateProps, nextState: IntermediateState): boolean {
-        const { whichGraph, source } = nextProps;
-        const beforeGraph = this.props.whichGraph;
-        if (whichGraph === '4') {
-            const { isFilter, length, clickCounts } = nextState;
-            const beforeLength = this.state.length;
-            const beforeSource = this.props.source;
-            const beforeClickCounts = this.state.clickCounts;
+    // will add it later. [graph is not render when add this component]
+    // shouldComponentUpdate(nextProps: IntermediateProps, nextState: IntermediateState) {
+    //     const { whichGraph, source } = nextProps;
+    //     const beforeGraph = this.props.whichGraph;
+    //     if (whichGraph === '4') {
+    //         const { isFilter, length, clickCounts } = nextState;
+    //         const beforeLength = this.state.length;
+    //         const beforeSource = this.props.source;
+    //         const beforeClickCounts = this.state.clickCounts;
 
-            if (isFilter !== this.state.isFilter) {
-                return true;
-            }
+    //         if (isFilter !== this.state.isFilter) {
```

Comment:
why annotate these lines?

Reply:
// will add it later. [graph is not render when add this component]
So... will delete this notes. will support this function after that.